app/helpers/application_helper/button/host_feature_button.rb

```diff
@@ -1,15 +1,8 @@
 class ApplicationHelper::Button::HostFeatureButton < ApplicationHelper::Button::GenericFeatureButton
 
   def visible?
-    # TODO: @feature.nil? || @record.nil? || @record.supports?(@feature)
-    # TODO: ensure start stop are supported in openstack
     unless @feature.nil? || @record.nil?
-      if @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager)
-        return true if %w[start stop].include?(@feature.to_s)
-        return false
-      end
-
-      return @record.supports?(@feature) if @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager::Host)
+      return @record.supports?(@feature)
     end
     true
```

Comment:
I think this simplifies to:

```
    return @record.supports?(@feature) if @record && @feature

    true
```

Reply:
> I don't have a record, so, yes, it's visible

I wonder if this is the list-view case where you haven't selected anything yet?  (just guessing would like UI team to chime in)